core-plugins/src/test/java/co/cask/hydrator/plugin/batch/action/HDFSFileMoveActionTestRun.java

```diff
@@ -364,8 +366,8 @@ public void testHDFSDeleteDirectoryAction() throws Exception {
     manager.start();
     manager.waitForFinish(3, TimeUnit.MINUTES);
 
-    Assert.assertFalse(fileSystem.exists(new Path(outputDir.toUri().toString() + "/dir1/source/test.txt")));
-    Assert.assertFalse(fileSystem.exists(new Path(outputDir.toUri().toString() + "/dir1/source/test2.txt")));
-    Assert.assertFalse(fileSystem.exists(new Path(outputDir.toUri().toString() + "/dir1/source/dir2/test2.txt")));
+    Assert.assertFalse(fileSystem.isDirectory(new Path(outputDir.toUri().toString() + "/dir1/source")));
```

Comment:
o it's asserting false. How come its asserting false for isDirectory() instead of exists()?